dotcom-rendering/src/web/browser/islands/doHydration.ts

```diff
@@ -26,7 +26,15 @@ export const doHydration = (name: string, data: any, element: HTMLElement) => {
 		`../../components/${name}.importable`
 	)
 		.then((module) => {
-			hydrate(h(module[name], data), element);
+			const clientOnly = element.getAttribute('clientOnly') === 'true';
```

Comment:
We can do this in another PR but ideally this value would be passed in as a param, from init. Just in terms of following the existing patterns